contents/docs/deployment/deploy-render.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Deploying to Render
+sidebar: Docs
+showTitle: true
+---
+
+## Why Render
+
+You can use Render's one-click deploy button to get Posthog up and running in minutes!
+
+[Render](https://render.com) is a modern cloud platform to build and run all your apps and websites with free SSL, a global CDN, private networks and auto deploys from Git.
+
+Every service on Render includes:
+- No limits on simultaneous connections or connection duration.
+- No forced restarts.
+- Full support for Docker containers.
+- Automatic [pull request previews](https://render.com/docs/pull-request-previews).
+- Infrastructure as Code with [`render.yaml`](https://render.com/docs/infrastructure-as-code).
+- [Fully managed PostgreSQL](https://render.com/docs/databases).
+- [Private networking](https://render.com/docs/private-services), load balancing, and service discovery.
+- HTTP health checks and [zero downtime deploys](https://render.com/docs/zero-downtime-deploys).
+- Free, unlimited [custom domains](https://render.com/docs/custom-domains) and [teams](https://render.com/docs/teams).
+- Persistent storage with [Render Disks](https://render.com/docs/disks).
+
+## Step By Step Installation
+
+[![Deploy to Render](https://render.com/images/deploy-to-render-button.svg)](https://render.com/deploy?repo=https://github.com/render-examples/posthog)
+
+1. Click Deploy to Render above.
+
+2. Follow the prompts to create an account if you haven't already and approve the creation of your Posthog services.
+
+    `Note: Because Posthog requires a higher tier plan, you will be asked to add a card. Charges for services will not be incurred until your free trial has expired.`
```

Comment:
I'd recommend using a slightly less passive voice: 'You won't be billed for your services until your trial ends'.

Reply:
```suggestion
    `Note: Because Posthog requires a higher tier plan, you will be asked to add a card. You won't be charged for your services until your trial ends.`
```